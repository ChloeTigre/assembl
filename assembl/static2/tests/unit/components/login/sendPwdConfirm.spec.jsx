import React from 'react';
import renderer from 'react-test-renderer';

import SendPwdConfirm from '../../../../js/app/components/login/sendPwdConfirm';

describe('SendPwdConfirm component', () => {
  it('should match the snapshot', () => {
    const component = renderer.create(<SendPwdConfirm />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});