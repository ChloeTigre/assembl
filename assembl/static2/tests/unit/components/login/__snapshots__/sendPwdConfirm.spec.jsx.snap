// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SendPwdConfirm component should match the snapshot 1`] = `
<div
  className="login-view"
>
  <div
    className="box-title"
  >
    <span
      className={undefined}
      style={undefined}
    >
      Password change requested
    </span>
  </div>
  <div
    className="box"
  >
    <form
      className="resendPwd"
      method="POST"
    >
      <div
        className="form-group"
      >
        <span
          className={undefined}
          style={undefined}
        >
          We have sent you an email with a temporary connection link. Please use that link to log in and change your password. Make sure to check your spam folder if an email is not in your inbox
        </span>
      </div>
    </form>
  </div>
</div>
`;
